### PR TITLE
Normalise parentheses

### PR DESCRIPTION
**Pull request by thewilkybarkid**

### CODE REVIEW

File: tests/cases/php/parentheses-new [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---DESCRIPTION---
+New instances must use parentheses
+---CONTENTS---
+<?php
```
Comment (giorgiosironi) [inline tests/cases/php/parentheses-new]: ` declare(strict_types=1);` missing from `---CONTENTS---`?

File: tests/cases/php/parentheses-useless [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+declare(strict_types=1);
+
+$foo = $bar !== null ? true : false;
+$baz = $qux ? 1 : 0;
+$quux = (true || false) ? 0 : 1;
+
+switch ($quuz) {
+    case true:
+    case false:
+        return $corge / (100 + $grault);
```
Comment (giorgiosironi) [inline tests/cases/php/parentheses-useless]: I'm curious about the effect on `(60 * 60 * 24) * $days` (or similar) which has unnecessary parentheses but can make the calculation more clear

Comment (thewilkybarkid) [reply tests/cases/php/parentheses-useless]: All those examples are unaffected.

(`(60 * 60 * 24 * $days)` is also unaffected when it _is_ useless though.)

**Comment by thewilkybarkid** [comment]
Build problem is addressed in https://github.com/libero/php-coding-standard/pull/37.

**Pull request merged**